cli.js

```diff
@@ -78,6 +87,6 @@ process.stdin.on('keypress', (ch, key) => {
 		}
 
 		prevResult = emojis;
-		logUpdate(`${pre}${chalk.bold(query.join(''))}\n${emojis}`);
+		logUpdate(`${pre}${chalk.bold(query.join(''))}\n${emojis}\n`);
```

Comment:
@CodeDotJS I'm not arguing against your problem, just your solution. If you're gonna add a newline you need to add it to all the logging, otherwise it will jump.